.chloggen/datadogexporter-new-trace-configs.yaml

```
@@ -0,0 +1,20 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: datadogexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add trace configs compute_stats_by_span_kind and peer_service_aggregation
```

nit: Use the full (namespaced) name for these options
```suggestion
note: "Add `traces::compute_stats_by_span_kind` and `traces::peer_service_aggregation` configuration flags
```

It would also be nice to have a single sentence description of what the flags are about